libkbfs/data_types.go

```diff
@@ -218,14 +253,35 @@ func (r BlockRef) String() string {
 	return s
 }
 
+type blockDirectType int
```

Comment:
might end up having to export this, if `kbfstool` has to read or manipulate these values